webplugin/js/app/zendesk-chat-service.js

```diff
@@ -28,10 +28,29 @@ function ZendeskChatService() {
             return;
         }
         window.console.log("handleUserMessage: ", event);
-        zChat.sendChatMsg(event.message.message, function (err, data) {
-            window.console.log("zChat.sendChatMsg ", err, data)
-        });
-        // TODO look for attachment
+        if(event.message.contentType==0){ //if user sends normal message
+            zChat.sendChatMsg(event.message.message, function (err, data) {
+                window.console.log("zChat.sendChatMsg ", err, data)
```

Comment:
is this contentType same as the contentType used in kommunicate's intent payload?

Reply:
Updated as requested.